draft-ietf-dprive-dnsoquic.md

```diff
@@ -381,7 +382,7 @@ notified to the client by sending back a response with the Response Code set to
 SERVFAIL.
 
 If a server is incapable of sending a DNS response due to an internal error, it
-SHOULD issue a QUIC Stream Reset. The error code SHOULD be set to DOQ_INTERNAL_ERROR. The
+SHOULD issue a QUIC RESET_STREAM frame. The error code SHOULD be set to DOQ_INTERNAL_ERROR. The
```

Comment:
(In the vein of the STOP_SENDING discussion with an earlier reviewer.)

Reply:
Yes, good suggestion. We need to incorporate Martin's comment about the asymmetry between clients and servers.